config/test.exs

```diff
@@ -6,7 +6,10 @@ config :dotcom, :httpoison, HTTPoison.Mock
 
 config :dotcom, :cms_api_module, CMS.Api.Static
 config :dotcom, :mbta_api_module, MBTA.Api.Mock
-config :dotcom, :repo_modules, route_patterns: RoutePatterns.Repo.Mock
+
+config :dotcom, :repo_modules,
+  route_patterns: RoutePatterns.Repo.Mock,
+  predictions: Predictions.Repo.Mock
```

Comment:
Should alphabetize this list and the one above because they are going to be quite long.